go.mod

```diff
@@ -3,12 +3,14 @@ module github.com/containers/buildah
 go 1.12
 
 require (
+	github.com/BurntSushi/toml v0.3.1
 	github.com/VividCortex/ewma v1.1.1 // indirect
 	github.com/blang/semver v3.5.0+incompatible // indirect
 	github.com/checkpoint-restore/go-criu v0.0.0-20190109184317-bdb7599cd87b // indirect
 	github.com/containerd/console v0.0.0-20181022165439-0650fd9eeb50 // indirect
 	github.com/containerd/continuity v0.0.0-20181203112020-004b46473808 // indirect
 	github.com/containernetworking/cni v0.7.1
+	github.com/containers/conmon v2.0.1+incompatible
```

Comment:
Why do we need conmon?

Reply:
https://github.com/containers/buildah/pull/1858#discussion_r325685772
> I think we should import this from
> `containers/conmon/runner/config` as it's a default that also configures conmon

Do I need to import for these same configures from conmon?

Reply:
https://github.com/containers/buildah/pull/1858#discussion_r325685772
> I think we should import this from
> `containers/conmon/runner/config` as it's a default that also configures conmon

Do I need to import for these same configures?

Reply:
I would prefer that you steal them, and then we get conmon to grab them from here.